Penteract/Assets/Scripts/AIDuke.cpp

```diff
@@ -120,6 +122,7 @@ void AIDuke::Update() {
 				if (player) {
 					float3 dir = player->GetComponent<ComponentTransform>()->GetGlobalPosition() - ownerTransform->GetGlobalPosition();
 					dir.y = 0.0f;
+					dir.Normalize();
```

Comment:
Why do you normalize it? If you need this, please do it inside Orientation, because the normalization breaks the ShootAndMove behavior.

Reply:
Now normalizing inside Orientate method